src/sdk/conference/client.js

```diff
@@ -115,6 +115,7 @@ export const ConferenceClient = function(config, signalingImpl) {
   const participants = new Map(); // Key is participant ID, value is a Participant object.
   const publishChannels = new Map(); // Key is MediaStream's ID, value is pc channel.
   const channels = new Map(); // Key is channel's internal ID, value is channel.
+  let mainChannel = null;
```

Comment:
Could you please add more description about `mainChannel`? That would help developers understand our SDK only creates a single `PeerConnection` for a `ConferenceClient`. But they can create more if they want. Then it will have a `secondaryChannel` or a channel list.